flake.nix

```diff
@@ -602,30 +602,54 @@
             buildInputs = [ pkgs.bbe ];
           };
 
-      in
-      {
-        packages = {
-          default = fedimintd;
```

Comment:
bug in the past code: we were not setting git hash for `fedimintd` built using `nix build .#`